global/css/_global.css

```diff
@@ -204,21 +204,43 @@ svg {
 
 .component-slider-single.component-slider-peek .swiper-slide-prev > *,
 .component-slider-single.component-slider-peek .swiper-slide-next > * {
-  @apply su-opacity-[.27] md:su-opacity-100 su-pointer-events-none md:su-pointer-events-auto;
+  @apply su-opacity-25 md:su-opacity-100 su-pointer-events-none md:su-pointer-events-auto;
```

Comment:
I made a very small update here for all Carousels (the ones that shows a peek of previous and next slides). Instead of custom TW class `su-opacity-[.27]`, I just use the out of the box `su-opacity-25` instead. I can't imagine the 2% opacity would make a huge difference so just trying to stick to out of the box classes as much as possible 😉